packages/synapse-react-client/src/style/components/_feed-cards.scss

```diff
@@ -33,7 +33,7 @@
       padding: 6px 25px;
       cursor: pointer;
       border: 1px solid transparent;
-      border-radius: 24px;
+      border-radius: 3px;
```

Comment:
Update the MailChimpSubscribe button in RssFeedCards to match other button border-radius